src/shared/iterate-emitter.js

```diff
@@ -1,8 +1,9 @@
-// @ts-nocheck
 /**
  * @module shared
  */
 
+import { EventEmitter } from 'events';
+
 /**
  * @param {EventEmitter} emitter
```

Comment:
We can avoid creating an unused binding like this:

```suggestion
/**
 * @param {import("events").EventEmitter} emitter
```

That way, any linting process [we might add](https://github.com/w3c/aria-at-automation-harness/issues/59) doesn't necessarily need to be aware of JSDoc.